configure.ac

```diff
@@ -4039,8 +4039,8 @@ if test "$ENABLED_SP_ASM" = "yes"; then
       ENABLED_SP_ARM_THUMB_ASM=yes
     else
       if test $host_alias = "cortex"; then
-        AM_CFLAGS="$AM_CFLAGS -DWOLFSSL_SP_ARM_CORTEX_ASM"
-        AM_CCASFLAGS="$AM_CCASFLAGS -DWOLFSSL_SP_ARM_CORTEX_ASM"
+        AM_CFLAGS="$AM_CFLAGS -DWOLFSSL_SP_ARM_CORTEX_M_ASM -mcpu=cortex-m4"
```

Comment:
Does it make sense to assume `cortex-m4` here? What about the `cortex-m3` or other M variants?

Reply:
gcc won't compile assembly code unless something is chosen.
cortex-m4 is what I'm testing and users will most likely have there own user_settings.h.